README.md

```diff
@@ -1,292 +1,126 @@
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- FUTURE FEATURES -->
-
-## 🔭 Future Features <a name="future-features"></a>
-
-> Describe 1 - 3 features you will add to the project.
-
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+## Future Features <a name="future-features"></a>
+
+- Add a graphical user interface.
+- Add a database to store data.
+- Add a login system for the librarian.
+- Add a search feature for books and students.
+- Add an option to sort books by title, author, genre, or publication date.
+- Add an option to sort students by name, grade, or age.
+- Add books classification using the Dewey Decimal System or the Library of Congress Classification System.
+- Add a feature to edit and delete books and students.
+- Add a feature to add multiple copies of the same book.
+- Add a feature to add multiple authors for the same book.
+- Add a feature to add multiple genres for the same book.
+- Add a feature that represents the borrowing history of a given student.
+- Add a feature that represents the borrowing history of a given book.
+- Add a feature for the librarian to add a fine for late returns.
+- Add a feature for the librarian to add a fine for lost books.
+- Add a feature for the librarian to add a fine for damaged books.
+- Add a feature that represents the book status (available, borrowed, lost, damaged).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- CONTRIBUTING -->
-
-## 🤝 Contributing <a name="contributing"></a>
+## Contributing <a name="contributing"></a>
 
 Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/mohamedSabry0/school_library/issues).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- SUPPORT -->
-
-## ⭐️ Show your support <a name="support"></a>
+## Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- ACKNOWLEDGEMENTS -->
-
-## 🙏 Acknowledgments <a name="acknowledgements"></a>
-
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
+If you like this project give it a star.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- FAQ (optional) -->
+## Acknowledgements <a name="acknowledgements"></a>
 
-## ❓ FAQ (OPTIONAL) <a name="faq"></a>
-
-> Add at least 2 questions new developers would ask when they decide to use your project.
-
-- **[Question_1]**
-
-  - [Answer_1]
-
-- **[Question_2]**
-
-  - [Answer_2]
+- Thanks to Microverse for encouraging us to do this project.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LICENSE -->
-
-## 📝 License <a name="license"></a>
+## License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Please kindly fix the MIT license broken link, having broken link in README don't look more professional 😁